docs/requirements_docs.txt

```diff
@@ -1,4 +1,3 @@
 sphinx
-sphinx-jsonschema
```

Comment:
I had to remove this package, as it's under GPL 2.0, which is not compatible with Apache 2, even as an imported library.

In addition, it does not seem to be required for generating the documentation.

Reply:
Cool, if not required, we should remove dependencies 👍